source/installation/system-requirements.txt

```diff
@@ -0,0 +1,73 @@
+.. _rm-system-requirements:
+
+===================
+System Requirements
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page outlines the system requirements for using Relational 
+Migrator. For additional details on where to run Relation Migrator, see 
+:ref:`rm-colocation`.
+
+Hardware
+--------
+
+You can run Relational Migrator in multiple ways:
+
+- Computer (can be a local desktop/labtop)
+- Remote server (EC2, Azure VM, or Google compute engine instance)
```

Comment:
Can we add a note that you must control RM through a web browser, which requires either:
 - Loading a browser on the machine running RM (locally or using Windows RDP or similar)
 - Accessing remotely, which requires configuring RM in Unattended mode